src/core.c/Rakudo/Internals.rakumod

```diff
@@ -1026,6 +1026,10 @@ my class Rakudo::Internals {
         #END leap-second-posix
     );
 
+    method current-offset-nanos() {
```

Comment:
Feels to me this calculation should only be done once at installation?  So prefix with `BEGIN`

Reply:
From my vague recollection, is that TimToady wanted astronomers to be able to use what is now Raku